src/related-items-and-comparisons/OutfitList.jsx

```diff
@@ -36,9 +43,14 @@ const OutfitList = (props) => {
       }
     }
     if (index > -1) {
-      updateOutfit(prevState => outfit.splice(index, 1));
-      updateOutfitIDs(prevState => outfitIDs.splice(index, 1));
+      // updateOutfit(prevState => outfit.splice(index, 1));
+      // updateOutfitIDs(prevState => outfitIDs.splice(index, 1));
```

Comment:
Use `slice()` first to clone the array. That way you're not mutating the original one.

Reply:
i didnt see this and just ended up making a new array and doing something different, slicing then splicing wouldve been faster, ill comment that in for a possible refactor later

Reply:
Actually, I like the way you did it a lot better than a slice-and-splice! It's really clever. Like a specialized `.filter()`.